### ISSUE TITLE
Update to newer Compose file version

### ISSUE THREAD

**Issue by strogonoff** [ietf-tools/bibxml-service#309]
### Description

Currently we use 3.3, but we could probably use 3.9+.

Perhaps @kesara could outline the possibility of this?

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by kesara** [comment]
Yes, it was downgraded because of the docker engine & docker-compose version limitation in the server which runs openSUSE Leap.
